bullet-featherstone/src/KinematicsFeatures.cc

```diff
@@ -113,10 +113,17 @@ FrameData3d KinematicsFeatures::FrameDataRelativeToWorld(
   FrameData data;
   if (model && model->body)
   {
-    data.pose = convert(model->body->getBaseWorldTransform());
-    if (!isModel)
-      data.pose = data.pose * model->baseInertiaToLinkFrame;
-    if (isCollision)
+    data.pose = convert(model->body->getBaseWorldTransform())
+        * model->baseInertiaToLinkFrame;
+    if (isModel)
+    {
+      data.pose = model->modelToRootLinkTf.inverse() * data.pose;
+      if (model->isNestedModel)
+      {
+        data.pose = data.pose * model->nestedModelFromRootModelTf;
+      }
+    }
+    else if (isCollision)
```

Comment:
This not really part of this PR, but I was wondering why whe need to handle `isCollision` here. Isn't it handled above?